### DISCUSSION TITLE
Stats API: Include imported GA data

### DISCUSSION THREAD

**Discussion by bogplau** [plausible/analytics#2620] [Anything Else]
The timeseries API endpoint does not work for data imported via Google Analytics. Is there a way to merge that data in timeseries API endpoint like the dashboard does? If not, could a parameter be added to do so?

received via email

**Comment by siebird** [comment]
I can see this being a critical feature with Google Analytics sunsetting Universal Analytics in a few months (July, 1st 2023)

**Comment by metmarkosaric** [comment]
**Update:** You can now export your imported data using our regular export methods (CSV and stats API). See how:

CSV export: https://plausible.io/docs/export-stats#export-specific-time-ranges-or-audience-segments
Stats API: https://plausible.io/docs/stats-api#imported-stats
